clamav/client.py

```diff
@@ -76,9 +77,16 @@ def scan_container_image(self, image_reference: str):
         '''
         logger.debug(f'scanning container image {image_reference}')
         for content, path in iter_image_files(image_reference):
-            scan_result = self.sse_scan(content)
-            if not scan_result.malware_detected():
-                continue
-            else:
-                return scan_result, path
-        return ClamAVScanResult({'malwareDetected':False}), ''
+            try:
+                scan_result = self.sse_scan(content)
+                if not scan_result.malware_detected():
+                    continue
+                else:
+                    yield (scan_result, path)
+            except ClamAVError as e:
+                if e.error_code() == 422:
```

Comment:
could you at least define a module variable with a more expressive name for this integer constant (and use it here in lieu of the literal)?